examples/abitest/README.md

```diff
@@ -0,0 +1,21 @@
+# ABI Test
```

Comment:
Should this be split into a different PR? It seems weird to add it here to support tests that are disabled and dependent on future work on reproducible builds.

Reply:
Update:
Without checking downgrades of public key labels tests are now working fine
I think testing the IFC itself should be done as a separate test